bintray.gradle

```diff
@@ -35,7 +35,7 @@ ext {
     siteUrl = 'https://github.com/cebroker/CustomComponents'
     gitUrl = 'https://github.com/cebroker/CustomComponents.git'
 
-    libraryVersion = '0.42.0'
+    libraryVersion = '0.43.0'
```

Comment:
`'0.43.0'` is the current version, use `'0.44.0'`

Reply:
Ok